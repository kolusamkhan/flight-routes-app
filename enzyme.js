const Adapter = require('enzyme-adapter-react-17-updated')

require('enzyme').configure({adapter: new Adapter()})