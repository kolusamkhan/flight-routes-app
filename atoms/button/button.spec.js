import 'jsdom-global/register';
import * as React from 'react';
import expect from 'expect';
import { shallow, mount, render } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Button from './index';

describe('Atom', () => {
  describe('Button', () => {
    it('should render without throwing an error', function () {
      const wrapper = shallow(<Button />);
      expect(shallowToJson(wrapper)).toMatchSnapshot();
    })
    
    it('should render type as button', function () {
        const wrapper = mount(<div><Button title="click me!" type="button" onClick={()=>{}} /></div>);
        expect(wrapper.find('button'));
      })
  })  
})