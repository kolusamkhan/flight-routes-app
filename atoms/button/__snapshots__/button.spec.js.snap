// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atom Button should render without throwing an error 1`] = `
<button
  type="button"
>
   
</button>
`;
